db/default_messages/upcoming_checkout_email

```diff
@@ -0,0 +1,8 @@
+Dear @user@,\n\n
```

Comment:
Note that these `\n`s don't actually get parsed correctly (not that it's critical because they're just seed data, but it's an issue nonetheless), both here and in the other default_messages. We should fix them while we're here. Also, note that there's a typo in the start of `upcoming_checkin_email` (`Return_date@` ==> `@return_date@`).